crates/rpc/rpc-eth-api/src/helpers/state.rs

```diff
@@ -134,25 +134,26 @@ pub trait EthState: LoadState + SpawnBlocking {
         &self,
         address: Address,
         block_id: BlockId,
-    ) -> impl Future<Output = Result<Account, Self::Error>> + Send {
+    ) -> impl Future<Output = Result<Option<Account>, Self::Error>> + Send {
         self.spawn_blocking_io(move |this| {
             let state = this.state_at_block_id(block_id)?;
 
-            let account = state
-                .basic_account(address)
-                .map_err(Self::Error::from_eth_err)?
-                .unwrap_or_default();
-            let balance = account.balance;
-            let nonce = account.nonce;
-            let code_hash = account.bytecode_hash.unwrap_or(KECCAK_EMPTY);
+            let account = state.basic_account(address).map_err(Self::Error::from_eth_err)?;
+            if let Some(account) = account {
+                let balance = account.balance;
+                let nonce = account.nonce;
+                let code_hash = account.bytecode_hash.unwrap_or(KECCAK_EMPTY);
 
-            // Provide a default `HashedStorage` value in order to
-            // get the storage root hash of the current state.
-            let storage_root = state
-                .hashed_storage_root(address, Default::default())
-                .map_err(Self::Error::from_eth_err)?;
+                // Provide a default `HashedStorage` value in order to
+                // get the storage root hash of the current state.
+                let storage_root = state
+                    .hashed_storage_root(address, Default::default())
+                    .map_err(Self::Error::from_eth_err)?;
```

Comment:
fix this whitespace pls, try command `make lint`

Reply:
nvm, duplicate of https://github.com/paradigmxyz/reth/pull/10402/files#r1723751263